openstack/master/main.tf

```diff
@@ -49,6 +49,18 @@ resource "null_resource" "generate-inventory" {
 
 }
 
+# Upload all additional public keys to master node
```

Comment:
Oh sorry, I just looked at the diff. Maybe we could add this directory to the KubeNow root, with a README.md inside?